### PR TITLE
Some renames

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
### Context

We are building up a glossary of terms and I'm picking off some quick wins in the UI

### Changes proposed in this pull request

A few renames (see commits) that cause a lot of code changes.

### Guidance to review

I recommend looking at the individual commits rather than the file diffs. Apart from the .cshtml, everything is auto-refactored.

### CODE REVIEW

File: ManageCoursesUi.Tests/Helpers/TestHelper.cs [hunk 1]
```diff
@@ -129,15 +118,15 @@ private static List<Course> GenerateCourseVariants(int count, bool happyPath, st
                         ProfpostFlag = "",
                         ProgramType = "SC",
                         Subjects = courseTitle + ",Secondary",
-                        CourseCode = ucasCode,
+                        CourseCode = instCode,
```
Comment (defong) [inline ManageCoursesUi.Tests/Helpers/TestHelper.cs]: Can i reject?

File: src/ui/Controllers/HomeController.cs [hunk 2]
```diff
@@ -43,7 +43,7 @@ public async Task<ActionResult> Index()
             var userOrganisations = orgs.ToList();
             if (userOrganisations.Count() == 1)
             {
-                return this.RedirectToAction("Show", "Organisation", new { ucasCode = userOrganisations[0].UcasCode });
+                return this.RedirectToAction("Show", "Organisation", new { instCode = userOrganisations[0].UcasCode });
```
Comment (defong) [inline src/ui/Controllers/HomeController.cs]: Can i reject?

File: src/ui/IManageApi.cs [hunk 3]
```diff
@@ -10,18 +10,17 @@ namespace GovUk.Education.ManageCourses.Ui
...
     public interface IManageApi
     {
-        Task<InstitutionCourses> GetCoursesByOrganisation(string ucasCode);
-        Task<Course> GetCourseByUcasCode(string instCode, string ucasCode);
-        Task<IEnumerable<UserOrganisation>> GetOrganisations();
-        Task<UserOrganisation> GetOrganisation(string instCode);
-        Task<UcasInstitution> GetUcasInstitution(string instCode);
-        Task LogAccessRequest(AccessRequest accessRequest);
+        Task<InstitutionCourses> GetCoursesOfInstitution(string instCode);
+        Task<Course> GetCourseByCode(string instCode, string courseCode);
```
Comment (defong) [inline src/ui/IManageApi.cs]: GetCourseByCourseCode please

File: src/ui/ViewModels/OrganisationViewModel.cs [hunk 4]
```diff
@@ -64,16 +64,16 @@ public OrganisationViewModel()
...
         {
             ucasInstitutionEnrichmentGetModel = ucasInstitutionEnrichmentGetModel ?? new UcasInstitutionEnrichmentGetModel { EnrichmentModel = new InstitutionEnrichmentModel() { AccreditingProviderEnrichments = new ObservableCollection<AccreditingProviderEnrichment>() } };
 
             var enrichmentModel = ucasInstitutionEnrichmentGetModel.EnrichmentModel;
 
             var result = new OrganisationViewModel
             {
-                InstitutionCode = ucasInstitution.InstCode,
-                InstitutionName = ucasInstitution.InstFull,
+                InstCode = institutionSummary.UcasCode,
```
Comment (defong) [inline src/ui/ViewModels/OrganisationViewModel.cs]: Can i reject?

File: ManageCoursesUi.Tests/Helpers/TestHelper.cs (hunk 1 above)
Comment (timabell) [reply ManageCoursesUi.Tests/Helpers/TestHelper.cs]: yes

File: src/ui/Controllers/HomeController.cs (hunk 2 above)
Comment (timabell) [reply src/ui/Controllers/HomeController.cs]: no

File: src/ui/ViewModels/OrganisationViewModelForAbout.cs [hunk 5]
```diff
@@ -55,7 +55,7 @@ public void MergeIntoEnrichmentModel(ref InstitutionEnrichmentModel enrichmentMo
             var aboutTrainingProviders = new ObservableCollection<AccreditingProviderEnrichment>(
                 AboutTrainingProviders.Select(x => new AccreditingProviderEnrichment
                 {
-                    UcasInstitutionCode = x.InstitutionCode,
+                    UcasInstitutionCode = x.InstCode,
```
Comment (defong) [inline src/ui/ViewModels/OrganisationViewModelForAbout.cs]: Can i reject?

File: src/ui/Controllers/HomeController.cs (hunk 2 above)
Comment (timabell) [reply src/ui/Controllers/HomeController.cs]: that UcasCode is from the api, needs fixing there

File: src/ui/ViewModels/SchoolViewModel.cs [hunk 6]
```diff
@@ -6,7 +6,7 @@ public class SchoolViewModel
     {
         public string LocationName { get; set; }
         public string Address { get; set; }
-        public string Code { get; set; }
+        public string CampusCode { get; set; }
```
Comment (defong) [inline src/ui/ViewModels/SchoolViewModel.cs]: Can i reject?

**Comment by timabell** [comment]
:+1:  caveat sau's feedback

File: src/ui/Views/Organisation/Show.cshtml [hunk 7]
```diff
@@ -73,7 +73,7 @@
         {
           <tr class="govuk-table__row">
             <td class="govuk-table__cell">
-              <a href="@Url.Action("show", "course", new {instCode = Model.InstitutionId, accreditingProviderId = course.AccreditingProviderId, ucasCode = course.CourseCode})" class="govuk-link">@course.Name (@course.CourseCode)</a>
+              <a href="@Url.Action("show", "course", new {instCode = Model.InstCode, accreditingInstCode = course.AccreditingProviderId, courseCode = course.CourseCode})" class="govuk-link">@course.Name (@course.CourseCode)</a>
```
Comment (defong) [inline src/ui/Views/Organisation/Show.cshtml]: can i reject `accreditingInstCode = course.AccreditingProviderId`

File: src/ui/Controllers/HomeController.cs (hunk 2 above)
Comment (defong) [reply src/ui/Controllers/HomeController.cs]: ok

File: ManageCoursesUi.Tests/Helpers/TestHelper.cs (hunk 1 above)
Comment (defong) [reply ManageCoursesUi.Tests/Helpers/TestHelper.cs]: good

**Comment by defong** [comment]
[rename all the things](https://trello.com/c/L9cCXGxO/523-rename-all-the-things)

**Comment by fofr** [comment]
Happy to see variants is going :)

File: src/ui/ViewModels/OrganisationViewModelForAbout.cs (hunk 5 above)
Comment (the-daniel-rothig) [reply src/ui/ViewModels/OrganisationViewModelForAbout.cs]: no that's in the Enrichment which comes from API

File: src/ui/ViewModels/OrganisationViewModel.cs (hunk 4 above)
Comment (the-daniel-rothig) [reply src/ui/ViewModels/OrganisationViewModel.cs]: No comes from API

File: src/ui/ViewModels/SchoolViewModel.cs (hunk 6 above)
Comment (the-daniel-rothig) [reply src/ui/ViewModels/SchoolViewModel.cs]: sry I don't get it -why?

File: src/ui/Views/Organisation/Show.cshtml (hunk 7 above)
Comment (the-daniel-rothig) [reply src/ui/Views/Organisation/Show.cshtml]: that's coming from API as well :)

File: src/ui/ViewModels/SchoolViewModel.cs (hunk 6 above)
Comment (the-daniel-rothig) [reply src/ui/ViewModels/SchoolViewModel.cs]: ooh b/c campus - sorry - yep will fix

**Pull request merged**